xml/System/GC.xml

```diff
@@ -1254,8 +1254,8 @@ This method is most useful in monitoring scenarios for measuring the difference
       <Docs>
         <param name="forceFullCollection">
           <see langword="true" /> to indicate that this method can wait for garbage collection to occur before returning; otherwise, <see langword="false" />.</param>
-        <summary>Retrieves the number of bytes currently thought to be allocated. A parameter indicates whether this method can wait a short interval before returning, to allow the system to collect garbage and finalize objects.</summary>
-        <returns>A number that is the best available approximation of the number of bytes currently allocated in managed memory.</returns>
+        <summary>Retrieves the heap size excluding fragmentation. For example if the total GC heap size is 100mb and fragmentation, ie, space taken up by free objects, takes up 40mb, this API would report 60mb. A parameter indicates whether this method can wait a short interval before returning, to allow the system to collect garbage and finalize objects.</summary>
+        <returns>A number that is the heap size excluding fragmentation.</returns>
```

Comment:
actually I think this should say "a number in bytes" to make it clear.

Reply:
I agree. Here's my suggestion:

```suggestion
        <returns>The heap size, in bytes, excluding fragmentation.</returns>
```